RuntimeUnityEditor/Utils/UI/InterfaceMaker.cs

```diff
@@ -1,8 +1,10 @@
 ﻿using System;
 using System.Reflection;
+using BepInEx;
```

Comment:
Probably missing a project ref, I didn't push project refs because I'm not sure that my compiling env is standard, I think it could be removed and probably unused, until eventually you migrate the plugin to .NET Standard, and check that it Compiles with Visual Studio Code on a Vanilla Windows 11 installation, would be the most optimal refs to have, having Visual Studio Community as a Requirement eats 30 GB of space.